hack/e2e.go

```diff
@@ -443,10 +446,15 @@ func finishRunning(stepName string, cmd *exec.Cmd) bool {
 	}(time.Now())
 
 	if err := cmd.Run(); err != nil {
-		log.Printf("Error running %v: %v", stepName, err)
-		return false
+		return fmt.Errorf("Error running %v: %v", stepName, err)
+	}
+	return nil
+}
+
+func doOrDie(msg string, err error) {
+	if err != nil {
+		log.Fatalf("error %s: %s", msg, err)
```

Comment:
Please don't do this. I know it sucks to repeat code, but please `log.Fatal` where the error is received and not in a utility method like this. What you have here will always die on line 456. https://play.golang.org/p/SdbuTBrh8P